generators/generator-bot-adaptive/package.json

```diff
@@ -21,14 +21,14 @@
   },
   "scripts": {
     "lint": "eslint -c .eslintrc.json .",
-    "test": "mocha tests"
+    "test": "mocha"
   },
   "dependencies": {
     "globby": "^11.0.2",
     "normalize-path": "^3.0.0",
     "runtypes": "~5.1.0",
     "uuid": "^8.3.1",
-    "yeoman-generator": "^1.1.1"
+    "yeoman-generator": "^2.0.5"
```

Comment:
Is the the furthest version bump we can take before there are breaking API changes?

Reply:
@joshgummersall it's not, but it's consistent with the version that is being used in the child generators. Per our discussion, figured dependency version bump should come in a different PR. Does that work?

Reply:
Ah yep, consistency is good. I figured everything was on the older version already. This is a major version bump though, is there anything like a changelog I can triple check to make sure all the adaptive bot Yeoman code is still correct?

Reply:
Just looking at https://github.com/yeoman/generator/releases/tag/v2.0.0 now, seems fine to me. We never used the functional extends pattern, nor do we use the install methods.